modules/python/init.zsh

```diff
@@ -93,7 +93,9 @@ if (( $+VIRTUALENVWRAPPER_VIRTUALENV || $+commands[virtualenv] )) && \
   VIRTUAL_ENV_DISABLE_PROMPT=1
 
   # Enable 'virtualenv' with 'pyenv'.
-  if (( $+commands[pyenv] && $+commands[pyenv-virtualenv-init] )); then
+  if (( $+commands[pyenv] )) && \
+     pyenv commands | command grep -q virtualenv-init
+  then
     eval "$(pyenv virtualenv-init -)"
     # Optionall activate 'virtualenvwrapper' with 'pyenv' is available.
     if (( $#commands[(i)pyenv-virtualenvwrapper(_lazy|)] )); then
```

Comment:
I'm gonna walk myself back on this since it seems like a safe assumption that if you installed the virtualenvwrapper plugin, you probably want to use it.